DeepSpace/src/main/cpp/Robot.cpp

```diff
@@ -10,10 +10,14 @@
 #include <frc/commands/Scheduler.h>
 #include <frc/smartdashboard/SmartDashboard.h>
 
+#include <iostream>
+
 ExampleSubsystem Robot::m_subsystem;
 OI Robot::m_oi;
 
 void Robot::RobotInit() {
+  std::cout << "RobotInit Started" << std::endl;
```

Comment:
Hey, consider "RobotInit Started\n", but this works for now as the `RobotInit` function is not time critical